src/modules/peek/Peek.UI/PeekXAML/MainWindow.xaml

```diff
@@ -23,6 +23,7 @@
             <KeyboardAccelerator Key="Right" Invoked="NextNavigationInvoked" />
             <KeyboardAccelerator Key="Down" Invoked="NextNavigationInvoked" />
             <KeyboardAccelerator Key="Escape" Invoked="EscKeyInvoked" />
+            <KeyboardAccelerator Key="W" Modifiers="Control" Invoked="CtrlWInvoked" />
```

Comment:
I’d rename the esc invoke to close invoked and have the two call the same method